### DISCUSSION TITLE
Response<BodyType> to BodyType converter

### DISCUSSION THREAD

**Discussion by dimitar-zabaznoski** [lejard-h/chopper#252] [Ideas]
Problem:
Instead of having `Response<BodyType>` as the return type of my endpoints, I would like to have a return type of just `BodyType` without being wrapped in `Response`

Current:
```
@Get(path: '/task')
Future<Response<Task>> getTask();
```

Proposed:
```
@Get(path: '/task')
Future<Task> getTask();
```

This could potentially be done with:
- another (final) converter that would convert Response<T> to whatever
- or, a dirtier approach, tampering options for the code generator

Is this something you consider useful and worthy for a future release?

**Comment by Guldem** [comment]
This was make possible by #545

**Reply by dimitar-zabaznoski** [reply]
Thank you for taking the time to address this.
